File: DatingApp.API/Models/Message.cs
using System;

namespace DatingApp.API.Models
{
    public class Message
    {
        public int Id { get; set; }
        public int SenderId { get; set; }
        public User Sender { get; set; }
        public int ReceiverId { get; set; }
        public User Receiver { get; set; }
        public string Content { get; set; }
        public bool IsRead { get; set; }
        public DateTime? DateRead { get; set; }
        public DateTime DateSent { get; set; }
        public bool SenderDeleted { get; set; }
        public bool ReceiverDeleted { get; set; }
    }
}

File: DatingApp-SPA/src/app/_models/message.ts
export class Message {
  id: number;
  senderId: number;
  senderKnownAs: string;
  senderPhotoUrl: string;
  receiverId: number;
  receiverKnownAs: string;
  receiverPhotoUrl: string;
  content: string;
  isRead: boolean;
  dateRead: Date;
  dateSent: Date;
}
